projects/js-packages/licensing/components/activation-screen-error/style.scss

```diff
@@ -0,0 +1,63 @@
+.activation-screen-error__message {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    max-width: 500px;
+    margin-top: calc(var(--spacing-base)*.5);
+
+    svg {
+        margin-left: -3px;
+    }
+
+    span {
+        font-size: 13px;
+        line-height: 20px;
+        font-weight: 500;
+        letter-spacing: -0.044em;
+    }
+}
+
+.activation-screen-error__message--error {
+    color: var(--jp-red);
+    svg {
+        fill: var(--jp-red);
+    }
+}
+
+.activation-screen-error__message--success {
+    color: var(--jp-green);
+    svg {
+        fill: var(--jp-green);
+    }
+}
+
+.activation-screen-error__info {
+    background-color: var(--jp-gray-0);
+    color: var(--jp-gray-80);
+    font-size: var(--font-body-small);
+    line-height: calc(var(--font-title-small) - 2px);
+    border: 1px solid #F0F0F0;
```

Comment:
We have a lot of grays already in the variables, I wonder if we really need to add a new one for this border specifically 🤔 Another thing that design would have a better feel for

Reply:
Yeah, I don't want --jp-gray-0 because it's the same color as the background color, and --jp-gray stands out a bit too dark, imo. But actually, --jp-green-0 (#f0f2eb) is very close to #F0F0F0, and with it being only a 1px border, it's almost impossible to tell the difference with the naked eye. It's the closest thing, so I'll use that! 👍😉

Reply:
Oh nice! I know I always try and suggest using existing variables, hopefully it's not too annoying 😅 

Thank you!